projects/packages/jetpack-mu-wpcom/src/features/help-center/class-help-center.php

```diff
@@ -52,11 +52,22 @@ private static function is_proxied() {
 	/**
 	 * Creates instance.
 	 *
-	 * @return \A8C\FSE\Help_Center
+	 * @return \A8C\FSE\Help_Center|void
```

Comment:
https://www.php.net/manual/en/language.types.void.php says `void` can't be part of a union type, and the return value is `null`. So it would probably be more accurate to do like this:
```suggestion
	 * @return \A8C\FSE\Help_Center|null
```

Or, since it doesn't look like the return value is ever actually used, you could just do `@return void` and never return anything.